content/doc/book/using/using-agents.adoc

```diff
@@ -0,0 +1,195 @@
+---
+layout: section
+title: Using Jenkins agents
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:toclevels: 4
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+= Jenkins agents architecture
+
```

Comment:
This text is ignored.  The level 1 heading is inserted by the framework using the value of `title` from the prolog.
```suggestion
```